packages/api/models/schemas.js

```diff
@@ -109,6 +109,12 @@ module.exports.collection = {
       description: 'The regex used to extract the granule id from granule id filenames',
       type: 'string'
     },
+    reportToEms: {
+      title: 'Report to EMS',
+      description: 'Indicates whether the collection will be reported to EMS',
+      type: 'boolean',
+      default: false
```

Comment:
Should this default to true? Wouldn't we want collections by default to be reported to EMS?